test/hotspot/jtreg/runtime/CommandLine/VMOptionWarning.java

```diff
@@ -44,6 +44,17 @@
  * @run driver VMOptionWarning Diagnostic
  */
 
+/* @test VMCompileCommandWarningDiagnostic
+ * @bug 8027314
+ * @summary Warn if compile command that is an alias for a diagnostic vm option is used and -XX:+UnlockDiagnosticVMOptions isn't specified.
+ * @requires vm.flagless
+ * @requires ! vm.debug
```

Comment:
```suggestion
 * @requires !vm.debug
```

Reply:
I think it's nicer indeed. I've kept it this way for consistency, but I don't mind making it nicer. Should I also fix the lines 29, 40 and 62 that have such a space?

Reply:
Yes, feel free to do that.